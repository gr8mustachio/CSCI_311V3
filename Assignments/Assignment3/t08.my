Q.size() = 1
Q.size() = 1
String s = abc, all permutations of s are:
a
Q.size() = 1
Q.size() = 1
Q.size() = 1
String s = abcd, all permutations of s are:
a
Q.size() = 1
Q.size() = 1
Q.size() = 1
Q.size() = 1
String s = zhewo, all permutations of s are:
z
