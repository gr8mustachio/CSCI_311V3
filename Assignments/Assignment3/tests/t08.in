findPerms
abc
findPerms
abcd
findPerms
zhewo
