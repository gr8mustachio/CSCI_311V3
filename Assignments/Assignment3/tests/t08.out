String s = abc, all permutations of s are:
abc
bac
bca
acb
cab
cba
String s = abcd, all permutations of s are:
abcd
bacd
bcad
bcda
acbd
cabd
cbad
cbda
acdb
cadb
cdab
cdba
abdc
badc
bdac
bdca
adbc
dabc
dbac
dbca
adcb
dacb
dcab
dcba
String s = zhewo, all permutations of s are:
zhewo
hzewo
hezwo
hewzo
hewoz
zehwo
ezhwo
ehzwo
ehwzo
ehwoz
zewho
ezwho
ewzho
ewhzo
ewhoz
zewoh
ezwoh
ewzoh
ewozh
ewohz
zhweo
hzweo
hwzeo
hwezo
hweoz
zwheo
wzheo
whzeo
whezo
wheoz
zweho
wzeho
wezho
wehzo
wehoz
zweoh
wzeoh
wezoh
weozh
weohz
zhwoe
hzwoe
hwzoe
hwoze
hwoez
zwhoe
wzhoe
whzoe
whoze
whoez
zwohe
wzohe
wozhe
wohze
wohez
zwoeh
wzoeh
wozeh
woezh
woehz
zheow
hzeow
hezow
heozw
heowz
zehow
ezhow
ehzow
ehozw
ehowz
zeohw
ezohw
eozhw
eohzw
eohwz
zeowh
ezowh
eozwh
eowzh
eowhz
zhoew
hzoew
hozew
hoezw
hoewz
zohew
ozhew
ohzew
ohezw
ohewz
zoehw
ozehw
oezhw
oehzw
oehwz
zoewh
ozewh
oezwh
oewzh
oewhz
zhowe
hzowe
hozwe
howze
howez
zohwe
ozhwe
ohzwe
ohwze
ohwez
zowhe
ozwhe
owzhe
owhze
owhez
zoweh
ozweh
owzeh
owezh
owehz
